pkg/cmd/project/connect-tenant/connect-tenant.go

```diff
@@ -0,0 +1,32 @@
+package connect
+
+import (
+	"fmt"
+	"github.com/MakeNowJust/heredoc/v2"
+	"github.com/OctopusDeploy/cli/pkg/cmd"
+	connectTenant "github.com/OctopusDeploy/cli/pkg/cmd/tenant/connect"
+	"github.com/OctopusDeploy/cli/pkg/constants"
+	"github.com/OctopusDeploy/cli/pkg/factory"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdConnect(f factory.Factory) *cobra.Command {
+	connectFlags := connectTenant.NewConnectFlags()
+	cmd := &cobra.Command{
+		Use:   "connect",
+		Short: "Connect a tenant to a project in Octopus Deploy",
+		Long:  "Connect a tenant to a project in Octopus Deploy",
+		Example: fmt.Sprintf(heredoc.Doc(`
+			$ %s project connect
+			$ %s project connect --project "Deploy web site" --environment "Production"
```

Comment:
Is this right? Should it be `--tenant`?

Reply:
Yes, this is correct but I think we should have an example that shows specifying both project and tenant.

Reply:
This should read, `%s project connect --tenant "Test Tenant" --environment "Production"`

Reply:
updated both commands to include `--tenant` in the example